src/BuildToolchains.jl

```diff
@@ -108,11 +108,17 @@ function toolchain_file(bt::CMake, p::AbstractPlatform; is_host::Bool=false)
     end
 end
 
+meson_c_args(p::AbstractPlatform) = "'-I/workspace/destdir/include'"
+meson_cxx_args(p::AbstractPlatform) = meson_c_args(p)
+meson_objc_args(p::AbstractPlatform) = meson_c_args(p) * ", '-x', 'objective-c'"
+meson_fortran_args(p::AbstractPlatform) = meson_c_args(p)
```

Comment:
Can we make these return arrays, then join them at the end?  I dislike string manipulation like this as it's not very flexible.